api/translate/translate/v1/translate.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto3";
+
+package translate.v1;
+
+import "google/api/annotations.proto";
+
+message Message {
+  string id = 1;
+  string translation = 2;
```

Comment:
We use **message** field in our project.

**Translation** field relates to **gotext** - when we transform to **gotext** format.

```suggestion
  string message = 2;
```